lbn.py

```diff
@@ -581,24 +595,30 @@ def add_weight(name, m):
         @tf.function
         def call(self, inputs):
             # forward to lbn.__call__
-            return self.lbn(inputs, features=self.feature_names)
+            return self.lbn(inputs, features=self._features)
     else:
         def call(self, inputs):
             # forward to lbn.__call__
-            return self.lbn(inputs, features=self.feature_names)
+            return self.lbn(inputs, features=self._features)
 
     def compute_output_shape(self, input_shape):
         return (input_shape[0], self.lbn.n_features)
 
     def get_config(self):
-        config = {
-            "n_particles": self.n_particles,
-            "boost_mode": self.boost_mode,
-            "features": self.feature_names,
-            "seed": self.seed,
-        }
-        base_config = super(LBNLayer, self).get_config()
-        return dict(list(base_config.items()) + list(config.items()))
+        config = super(LBNLayer, self).get_config()
+        config.update({
+            "n_particles": self.lbn.n_particles,
```

Comment:
```suggestion
            "n_particles": self.lbn.n_particles,
            "n_particles": self.lbn.n_restframes,
```
This could cause conflicts, when the user magically sets different values for this. Maybe, this should be a possibility in the future.

Reply:
Oh, thanks for spotting this. This should be "n_restframes".